apps/api/src/app/integrations/e2e/remove-integration.e2e.ts

```diff
@@ -0,0 +1,42 @@
+import { UserSession } from '@notifire/testing';
+import { expect } from 'chai';
+import { IntegrationRepository } from '@notifire/dal';
+
+describe('Get Integration - /integration (GET)', function () {
```

Comment:
```suggestion
describe('Delete Integration - /integration/:integrationId (DELETE)', function () {
```

Reply:
strange I'm sure I already added this change.
let me check it out